### DISCUSSION TITLE
Position Popup at Bottom of Screen with Bottom-Aligned URL Bar and Navigation Toolbox

### DISCUSSION THREAD

**Discussion by ashish-kus** [Godiesc/firefox-one#48] [General]
Hi everyone,

I'm trying to design a popup that appears at the bottom of the screen rather than the default top position. My setup includes both the URL bar and navigation toolbox positioned at the bottom of the screen, so ideally, the popup would appear just above these elements, maintaining a bottom-aligned layout.

I've looked through documentation and tried a few different approaches, but I haven't had any luck achieving this layout. Does anyone have experience with reversing the popup position or know of a way to get it to display at the bottom?

Any guidance or suggestions would be really appreciated. Thanks!

**Comment by Godiesc** [comment]
hi,
A great and professional guy helps with Firefox CSS in [this blog](https://lemmy.world/c/FirefoxCSS@fedia.io), you should try there for a solution of a difficult CSS behavior.
